rust/libceed/src/vector.rs

```diff
@@ -636,6 +636,40 @@ impl<'a> Vector<'a> {
         Ok(self)
     }
 
+    /// Compute y = alpha x + beta y for a pair of Vectors
+    ///
+    /// # arguments
+    ///
+    /// * `alpha` - first scaling factor
+    /// * `beta`  - second scaling factor
+    /// * `x`     - second vector, must be different than self
+    ///
+    /// ```
+    /// # use libceed::prelude::*;
+    /// # fn main() -> libceed::Result<()> {
+    /// # let ceed = libceed::Ceed::default_init();
+    /// let x = ceed.vector_from_slice(&[0., 1., 2., 3., 4.])?;
+    /// let mut y = ceed.vector_from_slice(&[0., 1., 2., 3., 4.])?;
+    ///
+    /// y = y.axpby(-0.5, 1.0, &x)?;
+    /// for (i, y) in y.view()?.iter().enumerate() {
+    ///     assert_eq!(*y, (i as Scalar) / 2.0, "Value not set correctly");
+    /// }
+    /// # Ok(())
+    /// # }
+    /// ```
+    #[allow(unused_mut)]
+    pub fn axpby(
```

Comment:
Can we add copy here too?

I think adding these functions in `python/ceed_vector.py` (and `python/tests/test-1-vector.py`) should be quick?